dhis-2/dhis-api/src/main/java/org/hisp/dhis/event/EventStatus.java

```diff
@@ -52,17 +53,9 @@ public int getValue() {
     return value;
   }
 
-  public static EventStatus fromInt(int status) {
-    for (EventStatus eventStatus : EventStatus.values()) {
-      if (eventStatus.getValue() == status) {
-        return eventStatus;
-      }
-    }
+  public static final Set<EventStatus> ALLOW_DATA_VALUES_STATUSES =
```

Comment:
wouldn't it make more sense to name it `ALLOWED_STATUSES`?
we are not referencing event data values here, are we?

Reply:
Anyway, is this used somewhere?

Reply:
`ALLOWED_STATUSES` is not true. These are the statuses that allow to have data values defined.
`ALLOWED_STATUSES` gets you the idea that those statuses are not allowed at all, which is not true.

Reply:
How about `STATUSES_ALLOWING_DATA_VALUES` or `STATUSES_WITH_DATA_VALUES`?